src/libraries/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -178,6 +178,15 @@ public void CreateEmpty()
             Assert.True(s_empty == ImmutableArray.Create(new int[0]));
         }
 
+        [Fact]
+        public void EmptyIsArrayEmpty()
+        {
+            var empty = ImmutableArray<int>.Empty;
+            var actualArray = Unsafe.As<ImmutableArray<int>, int[]>(ref empty);
```

Comment:
Looking at `ImmutableArrayExtensions`, `ToArray()` should return `ImmutableArray.Empty<int>()`.

```suggestion
            var actualArray = empty.ToArray();
```